### PR TITLE
Add performer-pytorch

### PR DESCRIPTION
**Pull request by maresb**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maresb** [comment]
Blocked by #20288 and #20289.

**Comment by maresb** [comment]
@conda-forge/help-python, ready for review (despite xfailing win-64 test).

This is a PyTorch noarch package, so win-64 fails simply because there's no PyTorch.

Generated by Grayskull.

File: recipes/performer-pytorch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - performer_pytorch
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/lucidrains/performer-pytorch
+  summary: Performer - Pytorch
```
Comment (BastianZim) [inline recipes/performer-pytorch/meta.yaml:38]:
```suggestion
  summary: An implementation of Performer, a linear attention-based transformer, in Pytorch
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just the summary could be slightly more descriptive, otherwise, it's fine.

**Comment by maresb** [comment]
Thanks @BastianZim for the suggestion! I have committed it, and now things are green (except of course win-64).

**Pull request merged**